src/components/z-date-picker/index.tsx

```diff
@@ -75,6 +125,73 @@ export class ZDatePicker {
     });
   }
 
+  setAriaOptions() {
```

Comment:
Questo metodo forse lo puoi strutturare suddivindendolo in funzioni più piccole, in base agli attributi che stai settando, una per ogni versione del picker, per esempio.